lib/v1.0/python/pprzlink/udp.py

```diff
@@ -0,0 +1,153 @@
+"""
+PPRZLINK UDP to Python interface
+"""
+from __future__ import absolute_import, division, print_function
+
+import threading
+import socket
+
+# used to run the test from this directory
+# any better implementation ?
+import os
+import sys
+PACKAGE_PARENT = '..'
+SCRIPT_DIR = os.path.dirname(os.path.realpath(os.path.join(os.getcwd(), os.path.expanduser(__file__))))
+sys.path.append(os.path.normpath(os.path.join(SCRIPT_DIR, PACKAGE_PARENT)))
```

Comment:
You can run the tests with `python -m pprzlink.serial` or `python -m pprzlink.udp` when you have set your PYTHONPATH